docs/spec/operators/cli.md

```diff
@@ -0,0 +1,35 @@
+---
+title: CLI
+---
+
+The CLI operator prints information to the terminal. When using `gadgetctl` or
+`kubectl-gadget`, this opereator takes the data coming from the server and
+prints it to the terminal.
```

Comment:
Maybe simpler:
The CLI operator is the default output operator - it prints all information and data from a gadget to the standard output (usually your terminal). It supports different output formats, depending on the gadget and the use case.